lib/model/profile_model.dart

```diff
@@ -58,7 +64,7 @@ class ProfileModel extends DataModel<ProfileModel> {
 
   /// convert two `List<String>` to `List<ProfileTag>`
   List<ProfileTag> _fromFirestoreTags(
-      List<String> tagList, List<String> tagContentList) {
+      List<dynamic> tagList, List<dynamic> tagContentList) {
```

Comment:
line 67 should be :
      List<String> tagList, List<String> tagContentList) {

and change line 110 to :
            ? _fromFirestoreTags(data['tags'] as List<String>, data['tag_contents'] as List<String>)

Reply:
resolve
with direct commit (!!?

Reply:
Get it, It is being fixed, and commit again.